<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>346</height>
   </rect>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>524</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>file</string>
    </property>
    <widget class="QMenu" name="menurecent_file">
     <property name="title">
      <string>recent file</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionclear"/>
    </widget>
    <addaction name="actionopen"/>
    <addaction name="actionsave"/>
    <addaction name="actionsave_as"/>
    <addaction name="separator"/>
    <addaction name="actionprint"/>
    <addaction name="actionprint_preview"/>
    <addaction name="actionprint_setting"/>
    <addaction name="menurecent_file"/>
    <addaction name="actionexit"/>
   </widget>
   <addaction name="menufile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionopen">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="text">
    <string>save</string>
   </property>
  </action>
  <action name="actionsave_as">
   <property name="text">
    <string>save as</string>
   </property>
  </action>
  <action name="actionprint">
   <property name="text">
    <string>print</string>
   </property>
  </action>
  <action name="actionprint_preview">
   <property name="text">
    <string>print preview</string>
   </property>
  </action>
  <action name="actionprint_setting">
   <property name="text">
    <string>print setting</string>
   </property>
  </action>
  <action name="actionexit">
   <property name="text">
    <string>exit</string>
   </property>
  </action>
  <action name="actionclear">
   <property name="text">
    <string>clear</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
